crates/bevy_ptr/src/lib.rs

```diff
@@ -397,13 +397,20 @@ impl<'a, T: ?Sized> From<&'a mut T> for PtrMut<'a> {
 }
 
 impl<'a> OwningPtr<'a> {
+    /// This exists mostly to reduce compile times;
+    /// code is only duplicated per type, rather than per function called.
+    fn make_internal<T>(temp: &mut ManuallyDrop<T>) -> OwningPtr<'_> {
+        // SAFETY: The value behind the pointer will not get dropped or observed later,
```

Comment:
```suggestion
        // SAFETY: Safety is unheld by the caller.
```

The following line should be deleted.

Reply:
I get making the function itself unsafe, but what value does this safety comment add? If safety is unheld, shouldn't we not be using unsafe code?

Reply:
Oops typo. It should be upheld

Reply:
Oh okay that makes sense haha

Reply:
The caller lets go of all safety constraints and becomes free.